requirements/edx/base.in

```diff
@@ -133,6 +133,7 @@ pyjwkest
 PyJWT>=1.6.3
 pylti1p3                            # Required by content_libraries core library to suport LTI 1.3 launches
 pymongo                             # MongoDB driver
+PyNaCl                              # User-friendly cryptography (wrapper and bindings for libsodium)
```

Comment:
Yep, currently can't run compile-requirements. I'll have to see if that's fixed by now.

Reply:
There is a GitHub action for the full upgrade.

Reply:
Yeah, running that against my branch now. I'm wondering if we should add a checkbox to that runner so people can select whether to upgrade or just recompile. It would make PRs less complicated. I'll take a note to look into this.

Reply:
(Specifically, there was something going on with a docs dependency <https://github.com/miyakogi/m2r/issues/66> -- it makes pip-compile fail on my Ubuntu laptop, but not on the Ubuntu Github Action. Go figure. I was hoping merging from master would help, but I think the issue is still unresolved.)